backend/config/curation-api.yml

```diff
@@ -272,6 +272,36 @@ paths:
           $ref: "#/components/responses/403"
         "404":
           $ref: "#/components/responses/404"
+    get:
+      summary: Return one or more datasets from a collection
+      description: >-
+        Return one or more datasets from a collection. If a dataset ID or a curator tag is provided as a query parameter
+        then only that dataset will be returned.
+      operationId: backend.corpora.lambdas.api.v1.curation.collections.collection_id.datasets.actions.get
+      tags:
+        - collection
+      parameters:
+        - $ref: "#/components/parameters/path_collection_id"
+        - $ref: "#/components/parameters/query_curator_tag"
+        - $ref: "#/components/parameters/query_dataset_id"
```

Comment:
Would now be an okay time to move to putting the identifier in the url path?
```
/v1/collections/{collection_id}/datasets/[rest/of/path/is/tag/or/id]
```

It is the most conventional approach for REST

Reply:
Although obviously it would apply to the other endpoints as well... so perhaps better put in a different PR

Reply:
I made the above comments thinking the endpoint was `/dataset` and not `/datasets`. See my comment on the main thread of this PR.